src/index.html

```diff
@@ -1,14 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+    content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+    href="./style.css">
+    <link rel="preconnect"
+    href="https://fonts.googleapis.com/">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com/"
+    crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Moyo logo"
+        class="logo__image">
```

Comment:
The same as above, check and fix it everywhere
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo logo"
          class="logo__image"
        >
```